.make/test.mk

```diff
@@ -167,7 +167,7 @@ test-integration-with-coverage: prebuild-check clean-coverage-integration migrat
 test-integration: prebuild-check migrate-database $(SOURCES)
 	$(call log-info,"Running test: $@")
 	$(eval TEST_PACKAGES:=$(shell go list ./... | grep -v $(ALL_PKGS_EXCLUDE_PATTERN)))
-	AUTH_DEVELOPER_MODE_ENABLED=1 AUTH_RESOURCE_DATABASE=1 AUTH_RESOURCE_UNIT_TEST=0 F8_LOG_LEVEL=$(F8_LOG_LEVEL) go test $(GO_TEST_INTEGRATION_FLAG) -vet off $(GO_TEST_VERBOSITY_FLAG) $(TEST_PACKAGES)
+	AUTH_DEVELOPER_MODE_ENABLED=1 AUTH_USER_DEACTIVATION_TESTING_MODE=FALSE AUTH_RESOURCE_DATABASE=1 AUTH_RESOURCE_UNIT_TEST=0 F8_LOG_LEVEL=$(F8_LOG_LEVEL) go test $(GO_TEST_INTEGRATION_FLAG) -vet off $(GO_TEST_VERBOSITY_FLAG) $(TEST_PACKAGES)
```

Comment:
minor: this is more about feature_flag rather than testing_flag so `AUTH_USER_DEACTIVATION_FEATURE_ENABLE` or something similar would be better.

Reply:
It's only temporary, i'll be removing this parameter in the next PR.